guide/.vuepress/clientAppEnhance.ts

```diff
@@ -1,4 +1,4 @@
-import { defineClientAppEnhance } from '@vuepress/client';
+import { defineClientAppEnhance } from "@vuepress/client";
```

Comment:
Why double quoted? Run prettier

Reply:
I think my prettiers config is messed up as I did.

Reply:
fixed and added a config from https://github.com/discordjs/discord.js/blob/main/.prettierrc.json in 5c6e0d2d25d69ab38c0299d94a84696f0b14a07b